lib/tool_shed/util/shed_index.py

```diff
@@ -66,8 +66,15 @@ def build_index(whoosh_index_dir, file_path, hgweb_config_dir, dburi, **kwargs):
             repo_index_writer.add_document(**repo)
 
             #  Tools get their own index
+            tool_index_writer.delete_by_term("repo_id", repo_id)
             for tool in tools_list:
-                tool_index_writer.add_document(**tool)
+                tool_contents = tool.copy()
+                # tool_id = tool_contents["id"]
```

Comment:
remove?

Reply:
Placed this change in the standing branch so I can merge these green tests https://github.com/galaxyproject/galaxy/pull/14583/commits/e9844a3d67ce81b27d2c52840c7f6e8339840383. Thanks so much for all the reviews @bgruening - I really appreciate it.